docs/README.rst

```diff
@@ -866,6 +866,84 @@ accessing the list of node and edge types:
    node_types  = types.all.nodes()
    edge_types  = types.all.edges()
 
+How to filter properties of elements of the same ontological type
```

Comment:
There's probably no need to duplicate documentation in both README and docs/README. I suggest we remove detailed doc from the root README and only use the docs one. The root README may be used for information interesting developers, the users being expected to rather read the doc on RTD.